### ISSUE TITLE
Problem with the mas install command

### ISSUE THREAD

**Issue by zoltanpesko** [ibm-mas/ansible-devops#1393]
22) Launch Install
```
If you are using storage classes that utilize 'WaitForFirstConsumer' binding mode choose 'No' at the prompt below
Wait for PVCs to bind? [y/n] n Traceback (most recent call last):
File "/opt/app-root/bin/mas-cli", line 54, in < module> app.install (argv|2:))
File "/opt/app-root/lib64/python3.9/site-packages/mas/cli/install/app.py", line 960, in install
installOpenShiftPipelines(self.dynamicClient)
File "/opt/app-root/lib64/python3.9/site-packages/mas/devops/tekton.py", line 75, in installOpenShiftPipelines
foundReadyWebhook = waitForDeployment(dynClient, namespace="openshift-pipelines", deploymentName="tekton-pipelines-webhook")
File "/opt/app-root/lib64/python3.9/site-packages/mas/devops/ocp.py", line 110, in waitForDeployment if deployment.status.readyReplicas > 0:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

**Comment by eclctcmd** [comment]
I'm running into a similar issue.

21) Launch Install
If you are using storage classes that utilize 'WaitForFirstConsumer' binding mode choose 'No' at the prompt below
Wait for PVCs to bind? [y/n] n
✅️ OpenShift Pipelines Operator is installed and ready to use
✅️ Namespace is ready (mas-demo-pipelines)
✅️ MAS CLI image deployment test completed
Fatal Exception: Could not execute kubectl command: task.tekton.dev/mas-devops-appconnect configured
task.tekton.dev/mas-devops-arcgis configured
task.tekton.dev/mas-devops-cert-manager configured
task.tekton.dev/mas-devops-common-services configured
task.tekton.dev/mas-devops-cos configured
task.tekton.dev/mas-devops-cp4d-service-update configured
task.tekton.dev/mas-devops-cp4d-service configured
task.tekton.dev/mas-devops-cp4d configured
task.tekton.dev/mas-devops-db2 configured
task.tekton.dev/mas-devops-eck configured
task.tekton.dev/mas-devops-gencfg-workspace configured
task.tekton.dev/gitops-bootstrap configured
task.tekton.dev/gitops-cis-compliance configured
task.tekton.dev/gitops-cluster configured
task.tekton.dev/gitops-cos configured
task.tekton.dev/gitops-cp4d-service configured
task.tekton.dev/gitops-cp4d configured
task.tekton.dev/gitops-db2u-database configured
task.tekton.dev/gitops-db2u configured
task.tekton.dev/gitops-delete-jdbc-config configured
task.tekton.dev/gitops-delete-kafka-config configured
task.tekton.dev/gitops-deprovision-app-config configured
task.tekton.dev/gitops-deprovision-app-install configured
task.tekton.dev/gitops-deprovision-cluster configured
task.tekton.dev/gitops-deprovision-cos configured
task.tekton.dev/gitops-deprovision-db2u-database configured
task.tekton.dev/gitops-deprovision-db2u configured
task.tekton.dev/gitops-deprovision-efs configured
task.tekton.dev/gitops-deprovision-kafka configured
task.tekton.dev/gitops-deprovision-mongo configured
task.tekton.dev/gitops-deprovision-rosa configured
task.tekton.dev/gitops-deprovision-suite-config configured
task.tekton.dev/gitops-deprovision-suite-idp-config configured
task.tekton.dev/gitops-deprovision-suite-objectstorage-config configured
task.tekton.dev/gitops-deprovision-suite-smtp-config configured
task.tekton.dev/gitops-deprovision-suite-watson-studio-config configured
task.tekton.dev/gitops-deprovision-suite-workspace configured
task.tekton.dev/gitops-deprovision-suite configured
task.tekton.dev/gitops-dro configured
task.tekton.dev/gitops-efs configured
task.tekton.dev/gitops-jdbc-config configured
task.tekton.dev/gitops-kafka-config configured
task.tekton.dev/gitops-kafka configured
task.tekton.dev/gitops-license configured
task.tekton.dev/gitops-mas-fvt-preparer configured
task.tekton.dev/gitops-mas-initiator configured
task.tekton.dev/gitops-mongo configured
task.tekton.dev/gitops-nvidia-gpu configured
task.tekton.dev/gitops-process-mongo-user configured
task.tekton.dev/gitops-rosa configured
task.tekton.dev/gitops-suite-app-config configured
task.tekton.dev/gitops-suite-app-install configured
task.tekton.dev/gitops-suite-certs configured
task.tekton.dev/gitops-suite-config configured
task.tekton.dev/gitops-suite-dns configured
task.tekton.dev/gitops-suite-idp-config configured
task.tekton.dev/gitops-suite-objectstorage-config configured
task.tekton.dev/gitops-suite-smtp-config configured
task.tekton.dev/gitops-suite-watson-studio-config configured
task.tekton.dev/gitops-suite-workspace configured
task.tekton.dev/gitops-suite configured
task.tekton.dev/mas-devops-grafana configured
task.tekton.dev/mas-devops-ibm-catalogs configured
task.tekton.dev/mas-devops-kafka configured
task.tekton.dev/mas-devops-mongodb configured
task.tekton.dev/mas-devops-must-gather configured
task.tekton.dev/mas-devops-nvidia-gpu configured
task.tekton.dev/mas-devops-ocp-verify-all configured
task.tekton.dev/mas-devops-ocp-verify configured
task.tekton.dev/mas-devops-ocs configured
task.tekton.dev/sls-registry-update configured
task.tekton.dev/mas-devops-sls configured
task.tekton.dev/mas-devops-suite-app-config configured
task.tekton.dev/mas-devops-suite-app-install configured
task.tekton.dev/mas-devops-suite-app-rollback configured
task.tekton.dev/mas-devops-suite-app-uninstall configured
task.tekton.dev/mas-devops-suite-app-upgrade configured
task.tekton.dev/mas-devops-suite-app-verify configured
task.tekton.dev/mas-devops-suite-certs configured
task.tekton.dev/mas-devops-suite-config configured
task.tekton.dev/mas-devops-suite-db2-setup-for-manage configured
task.tekton.dev/mas-devops-suite-dns configured
task.tekton.dev/mas-devops-suite-install configured
task.tekton.dev/mas-devops-suite-rollback configured
task.tekton.dev/mas-devops-suite-uninstall configured
task.tekton.dev/mas-devops-suite-upgrade configured
task.tekton.dev/mas-devops-suite-verify configured
task.tekton.dev/mas-devops-turbonomic configured
task.tekton.dev/mas-devops-uds configured
task.tekton.dev/mas-devops-update-configmap configured
task.tekton.dev/mas-devops-wait-for-configmap configured
task.tekton.dev/mas-devops-wait-for-tekton configured
pipeline.tekton.dev/mas-db2 configured
pipeline.tekton.dev/gitops-deprovision-cluster-after-deprovision unchanged
pipeline.tekton.dev/gitops-deprovision-cluster unchanged
pipeline.tekton.dev/gitops-deprovision-mas-cluster unchanged
pipeline.tekton.dev/gitops-deprovision-mas-deps-after-deprovision configured
pipeline.tekton.dev/gitops-deprovision-mas-deps configured
pipeline.tekton.dev/gitops-deprovision-mas-instance unchanged
pipeline.tekton.dev/gitops-mas-apps unchanged
pipeline.tekton.dev/gitops-mas-cluster unchanged
pipeline.tekton.dev/gitops-mas-fvt-preparer-pipeline unchanged
pipeline.tekton.dev/gitops-mas-initiator-after-provision unchanged
pipeline.tekton.dev/gitops-mas-instance unchanged
pipeline.tekton.dev/mas-install configured
pipeline.tekton.dev/gitops-provision-bootstrap-cluster-after-deprovision unchanged
pipeline.tekton.dev/gitops-provision-bootstrap-cluster unchanged
pipeline.tekton.dev/mas-rollback configured
pipeline.tekton.dev/mas-uninstall configured
pipeline.tekton.dev/mas-update configured
pipeline.tekton.dev/mas-upgrade configured
Error from server (BadRequest): error when creating "/opt/app-root/lib64/python3.9/site-packages/mas/cli/templates/ibm-mas-tekton.yaml": admission webhook "validation.webhook.pipeline.tekton.dev" denied the request: validation failed: non-existent variable in "$(params.github_pat)": spec.tasks[3].params[github_pat]

**Comment by durera** [comment]
```
File "/opt/app-root/lib64/python3.9/site-packages/mas/devops/ocp.py", line 110, in waitForDeployment if deployment.status.readyReplicas > 0:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

This is resolved here: https://github.com/ibm-mas/python-devops/pull/16

```
Error from server (BadRequest): error when creating "/opt/app-root/lib64/python3.9/site-packages/mas/cli/templates/ibm-mas-tekton.yaml": admission webhook "validation.webhook.pipeline.tekton.dev" denied the request: validation failed: non-existent variable in "$(params.github_pat)": spec.tasks[3].params[github_pat]
```

This is fixed here: https://github.com/ibm-mas/cli/pull/1185

Fixes for both are available when using the latest CLI version (10.8.0 at time of writing).  Please let us know if that version is not working for you/exhibits the same issues.
